SafeguardSessions/README.md

```diff
@@ -150,29 +156,25 @@ If a section does not have related content, it can be left out.
 
 Modifications to the Report Template must be listed in the appropriate sections, since they are part of the release.
 
-## Upgrade and refresh
+## Refreshing your preloaded data
+
+### To refresh your preloaded data
+1. Open your previously saved .pbix file in Power BI Desktop.
+2. In Power BI Desktop, click **Home > Refresh**.
+
+If an error occures after refreshing your data, discard your changes to keep your data intact.
+
+## Upgrading the Power BI Connector
 
 If you are a new user who haven't used the custom connector yet, follow the steps of [Installation and usage].
 
-You can upgrade to a newer version of the connector, if the version of your SPS is in the list of supported versions on the [Releases] page.
+You can upgrade to a newer version of the connector, if the version of your SPS is in the list of supported versions on the [Releases] page. If the upgraded connector introduces breaking changes according to our [Release policy], you need to refresh your preloaded data by following the steps described in [Refresh].
```

Comment:
From the custom connector side: there is no incremental refresh, so refresh will rewrite the whole dataset, regardless of the changes.
From the report template side: a new report template must be downloaded. If they want to use an old one, then they should modify it according to the changelog. I think we need a new subchapter for the report template.

Reply:
How about drawing the customer's attention first to the breaking change?

We could state here explicitly as it won't change in the future:
"... you can upgrade safely to a new connector if the major version number of the new connector does not differ from the previous. If it does, please view the Breaking changes section to be able to act upon it. ..."

Since each breaking change may require a unique way to handle it, we should rather write about the "how to handle" in the release notes in the Breaking changes section. So I don't think we need a new subchapter for the report template.

What do you think?

Reply:
Sounds good!

Reply:
Done.